Libraries/LibJS/Runtime/MathObject.cpp

```diff
@@ -28,13 +28,23 @@
 #include <AK/Function.h>
 #include <LibJS/Interpreter.h>
 #include <LibJS/Runtime/MathObject.h>
+#include <LibM/math.h>
 
 namespace JS {
 
 MathObject::MathObject()
 {
     put_native_function("abs", abs);
     put_native_function("random", random);
+
+    put_native_property("E", e_getter, nullptr);
+    put_native_property("LN2", ln2_getter, nullptr);
+    put_native_property("LN10", ln10_getter, nullptr);
+    put_native_property("LOG2E", log2e_getter, nullptr);
+    put_native_property("LOG10E", log10e_getter, nullptr);
+    put_native_property("PI", pi_getter, nullptr);
+    put_native_property("SQRT1_2", sqrt1_2_getter, nullptr);
+    put_native_property("SQRT2", sqrt2_getter, nullptr);
```

Comment:
We don't need callbacks for these, we can just `put` them directly:

```cpp
put("E", Value(M_E));
```

etc :)

Reply:
I knew this must be possible in a less verbose way! Thanks :)